src/components/explorer/IndividualMedicalActions.js

```diff
@@ -0,0 +1,324 @@
+                            {therapeuticRegimen?.external_reference?.id ?? EM_DASH}
+                        </div>
+                        <div>
+                            <strong>Reference:</strong>{" "}
+                            {therapeuticRegimen?.external_reference?.reference ?? EM_DASH}
+                        </div>
+                        <div>
+                            <strong>Description:</strong>{" "}
+                            {therapeuticRegimen?.external_reference?.description ?? EM_DASH}
+                        </div>
+                    </div>
+                }
+            </Descriptions.Item>
+            <Descriptions.Item label="Start Time">
+                {therapeuticRegimen?.start_time &&
+                    <TimeElement timeElement={therapeuticRegimen.start_time}/>
+                }
+            </Descriptions.Item>
+            <Descriptions.Item label="End Time">
+                {therapeuticRegimen?.start_time &&
+                    <TimeElement timeElement={therapeuticRegimen.end_time}/>
+                }
+            </Descriptions.Item>
+            <Descriptions.Item label="Status">
+                {therapeuticRegimen?.status ? therapeuticRegimen.status : EM_DASH}
+            </Descriptions.Item>
+        </Descriptions>
+    );
+};
+TherapeuticRegimen.propTypes = {
+    therapeuticRegimen: PropTypes.object,
+    resourcesTuple: PropTypes.array,
+};
+
+const MedicalActionDetails = ({medicalAction, resourcesTuple}) => {
+    const actionType = getMedicalActionType(medicalAction);
+
+    // The action is the only field always present, other fields are optional.
+    return (
+        <Descriptions bordered={true} column={1} size="small">
+            <Descriptions.Item label={actionType.name}>
+                {medicalAction?.procedure &&
+                    <Procedure procedure={medicalAction.procedure} resourcesTuple={resourcesTuple}/>
+                }
+                {medicalAction?.treatment &&
+                    <Treatment treatment={medicalAction?.treatment} resourcesTuple={resourcesTuple}/>
+                }
+                {medicalAction?.radiation_therapy &&
+                    <RadiationTherapy
+                        radiationTherapy={medicalAction.radiation_therapy}
+                        resourcesTuple={resourcesTuple}
+                    />
+                }
+                {medicalAction?.therapeutic_regimen &&
+                    <TherapeuticRegimen
+                        therapeuticRegimen={medicalAction.therapeutic_regimen}
+                        resourcesTuple={resourcesTuple}
+                    />
+                }
+            </Descriptions.Item>
+            <Descriptions.Item label="Adverse Events">
+                { Array.isArray(medicalAction?.adverse_events) ?
+                    medicalAction.adverse_events.map((advEvent, index) =>
+                        <OntologyTerm resourcesTuple={resourcesTuple} term={advEvent} key={index}/>)
```

Comment:
since this is just mapped to a bunch of ontology terms, there's a chance the spacing won't be correct here - not sure

Reply:
Added a `br` prop to `OntologyTerm` to render a conditional trailing `<br />` element to the span